class NotFounderError(Exception):
    pass


class DuplicateError(Exception):
    pass
